Features/Content/place.feature

```diff
@@ -11,3 +18,11 @@ Feature: Place
     Scenario: As a visitor, I can see the place list sorted
         Given I am on "/placelist/110/45.6152665/5.2234365/10000"
         Then I see "Mount Ventoux" text
+
+    Scenario: As a visitor, I can see the Mount Fuji page
+        Given I am on "/eZ-Mountains/Mount-Fuji"
+        Then I see "Mount Fuji" text
+        And I see "Mount Fuji, located on Honshu Island, is the highest mountain in Japan at 3,776.24 m (12,389 ft)." text
+        And I see the thumbnail for image with alternative text "mount fuji"
+        And Path for thumbnail image with alternative text "mount fuji" should match "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg"
+        And File "/var/ezdemo_site/storage/images/media/images/mount-fuji/552-1-eng-GB/mount-Fuji_imageplacesize.jpg" exists
```

Comment:
It is more ok, but the discuss me and @bdunogier had above was me trying to suggest ways to avoid even that, by having _something_ (you'r more deeply involved in this, so feel free to suggest other forms of verifying image and alternative text*) like 
`
And I see a valid thumbnail for image with alternative text "mount fuji"
`

Where "valid" would imply taking the path and doing a http call to verify it does not return 404.

\* Some input from @ezsystems/qa-team would be good here also